docs/csharp/language-reference/operators/pointer-related-operators.md

```diff
@@ -36,7 +36,7 @@ For information about pointer types, see [Pointer types](../../programming-guide
 > [!NOTE]
 > Any operation with pointers requires an [unsafe](../keywords/unsafe.md) context. The code that contains unsafe blocks must be compiled with the [`-unsafe`](../compiler-options/unsafe-compiler-option.md) compiler option.
 
-## Address-of operator &amp;
+## <a name="address-of-operator-amp"></a> Address-of operator &amp;
```

Comment:
(1) Shouldn't it be on its own line?
(2) Updated the bookmark name to match one used throughout the docs (see line 28 of this file)
```suggestion
<a name="address-of-operator-"></a>
## Address-of operator &amp;
```

Reply:
Another option is to try out the following H2 title:
```text
## Address-of operator &
```

Would it work? I guess it would generate the same bookmark text in both original and translated versions.

Reply:
I think that the solution proposed by @pkulikov should work, @v-maudel. That's the bookmark that's automatically generated by our builds, and also the bookmark that's referenced in three additional files.